src/components/TabGroup/__tests__/TabGroup.test.js

```diff
@@ -51,26 +51,78 @@ describe('TabGroup', () => {
         </TabGroup.Tab>
       </TabGroup>,
     ))
+
+  test('updates tab index on click', () => {
+    const onChange = jest.fn()
+    const onFirstTabClick = jest.fn()
+    const { getByText } = render(
+      <ThemeProvider theme={theme}>
```

Comment:
You can use the renderWithTheme helper for this 

eg: https://github.com/scaleway/scaleway-ui/blob/d60cdf2753dc817f3ee8ff0e369b46b5e0a40771/src/components/UnitInput/__tests__/index.js#L75

Reply:
Done 6aadc2a thanks.